backend_new/src/dtos/unit-accessibility-priority-types/unit-accessibility-priority-type.dto.ts

```diff
@@ -8,5 +9,6 @@ export class UnitAccessibilityPriorityType extends AbstractDTO {
   @IsString({ groups: [ValidationsGroupsEnum.default] })
   @IsDefined({ groups: [ValidationsGroupsEnum.default] })
   @MaxLength(256, { groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty()
   name: string;
```

Comment:
Same thing with unit type names. Can you make this an enum?

Reply:
updated